examples/tracing/docker-compose.yaml

```diff
@@ -0,0 +1,54 @@
+version: "3"
```

Comment:
Could we add all of these to the docker-compose.yml in the project root?

Reply:
hm, I think so. When I created these I had this tracing specific use case. I'll move these to the root in a different PR and review how it can integrate with the cli code or grafbase's docker image.